tests/pages/api/definePassengerType.test.ts

```diff
@@ -457,16 +457,43 @@ describe('definePassengerType', () => {
 
     it('should not save the passenger types for groups', async () => {
         const upsertPassengerTypeSpy = jest.spyOn(auroradb, 'upsertPassengerType');
+        const groupPassengerTypesAttribute: GroupPassengerTypesCollection = { passengerTypes: ['adult', 'child'] };
+        const groupSizeAttribute: GroupTicketAttribute = { maxGroupSize: '20' };
+        const mockPreviousPassengerTypeDetails: CompanionInfo[] = [
+            {
+                minNumber: '2',
+                maxNumber: '10',
+                ageRangeMin: '5',
+                ageRangeMax: '10',
+                proofDocuments: ['Membership Card', 'Student Card'],
```

Comment:
I don't think it actually affects the test, but users aren't given the option of specifying proofDocuments for adults, so might be more accurate to remove this property?

Reply:
👍